qds_sdk/commands.py

```diff
@@ -228,9 +228,9 @@ def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True):
         else:
             if fetch:
                 storage_credentials = conn.get(Account.credentials_rest_entity_path)
-                boto_conn = boto.connect_s3(aws_access_key_id=storage_credentials['storage_access_key'],
-                                            aws_secret_access_key=storage_credentials['storage_secret_key'],
-                                            security_token = storage_credentials['session_token'])
+                boto_conn = boto.connect_s3(aws_access_key_id=storage_credentials.get('storage_config').get('access_key'),
```

Comment:
Account apis still use only v1.2. So here we are making use of the more cloud agnostic API (/creds) instead of get_creds API. The api change will be pushed into production with r40. It might not work with api.qubole.com now.